apollo-router/src/apollo_studio_interop/mod.rs

```diff
@@ -356,33 +404,60 @@ fn format_selection_set(selection_set: &SelectionSet, f: &mut fmt::Formatter) ->
     }
 
     if !fields.is_empty() || !named_fragments.is_empty() || !inline_fragments.is_empty() {
-        fields.sort_by(|&a, &b| a.name.cmp(&b.name));
+        if is_enhanced(normalization_algorithm) {
```

Comment:
We'd have to pass around a boolean instead of the enum if we only wanted to call this once, and I figured it would be better to pass around the enum instead in case we add a new algorithm in the future. I'll change it to a macro though which should fix any performance concerns.

Reply:
macro vs function doesn't really make a performance difference (especially because the function is simple). It's better to use the function here